README.md

```diff
@@ -17,95 +17,92 @@ This card is available in [HACS](https://hacs.xyz/) (Home Assistant Community St
 |------|:----:|:-------:|:-----:|-------------|
-| height | number | 46 | v0.0.1 | Set the height of the bars.
-| round | number | 1 | v0.0.1 | Set the round radius for the bars.
-| spacing | number | 4 | v0.0.1 | Set the spacing between the bars.
-| amount | number | 36 | v0.0.1 | Set the number bars.
+| height | number | `46` | v0.0.1 | Set the height of the bars.
+| round | number | `1` | v0.0.1 | Set the round radius for the bars.
+| spacing | number | `4` | v0.0.1 | Set the spacing between the bars.
+| amount | number | `36` | v0.0.1 | Set the number of bars.
 
-### Color configuration
+### Color config
 
-**Parent key:** color
+**Parent key:** `color`
 
 | Name | Type | Default | Since | Description |
 |------|:----:|:-------:|:-----:|-------------|
-| ok | color | #45C669 | v0.0.1 | Set the ok color.
-| ko | color | #C66445 | v0.0.1 | Set the ko color.
-| half | color | #C6B145 | v0.0.1 | Set the half color.
-| none | color | #C9C9C9 | v0.0.1 | Set the none color.
-| title | color | grey | v0.0.2 | Set the title text color, title_adaptive_color must be false.
-| status | color | gray | v0.0.2 | Set the status text color, status_adaptive_color must be false.
-| icon | color | | v0.0.2 | Set the icon text color, icon_adaptive_color must be false.
-| tooltip | color | grey | v0.1.0 | Set the tooltip text color, tooltip_adaptive_color must be false.
+| ok | color | ![ ](https://dummyimage.com/20x10/45c669&amp;text=+) `#45C669` | v0.0.1 | Set the `ok` color.
+| ko | color | ![ ](https://dummyimage.com/20x10/c66445&amp;text=+) `#C66445` | v0.0.1 | Set the `ko` color.
+| half | color | ![ ](https://dummyimage.com/20x10/c6b145&amp;text=+) `#C6B145` | v0.0.1 | Set the `half` color.
+| none | color | ![ ](https://dummyimage.com/20x10/c9c9c9&amp;text=+) `#C9C9C9` | v0.0.1 | Set the `none` color.
+| title | color | `grey` | v0.0.2 | Set the title text color, `title_adaptive_color` must be false.
+| status | color | `gray` | v0.0.2 | Set the status text color, `status_adaptive_color` must be false.
+| icon | color | | v0.0.2 | Set the icon text color, `icon_adaptive_color` must be set to `false`.
+| tooltip | color | `grey` | v0.1.0 | Set the tooltip text color, tooltip_adaptive_color must be false.
 
-### Show configuration
+### Show config
 
-**Parent key:** show
+**Parent key:** `show`
 
-| Name | Type | Default | Since | Description |
-|------|:----:|:-------:|:-----:|-------------|
-| header | boolean | true | v0.0.1 | Show the header.
-| title | boolean | true | v0.1.0 | Show the title.
-| icon | boolean | true | v0.0.1 | Show the icon.
-| status | boolean | true | v0.0.1 | Show the status.
-| timeline | boolean | true | v0.0.1 | Show the timeline.
-| footer | boolean | true | v0.0.1 | Show the footer.
-| average | boolean | true | v0.0.2 | Show the average.
+| Name | Default | Options | Description |
+|------:|:------:|:-----:|-------------|
+| header | `true` | `true`/`false` | Show/hide the header.
```

Comment:
Why did you removed versions here ?

Reply:
You mean `average`? It’s still there.

Reply:
Just ordered above `footer` ;)

Reply:
I mean that other config have these columns: `| Name | Type | Default | Since | Description |` but for this one you changed them for `| Name | Default | Options | Description |`. And the version column just disapeared ^^'

Reply:
Gotcha.